test/SlowTests/Issues/RavenDB_21525.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Linq;
+using Raven.Client.Documents.Operations.Configuration;
+using Raven.Server.Config;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_21525 : RavenTestBase
+{
+    public RavenDB_21525(ITestOutputHelper output) : base(output)
+    {
+    }
+
+
+    [RavenTheory(RavenTestCategory.Corax)]
+    [RavenData(SearchEngineMode = RavenSearchEngineMode.All)]
+    public async Task OrderByWithPaginationVsUnordered(Options options, int size = 10000)
```

Comment:
Why do we have a test from different PR https://github.com/ravendb/ravendb/pull/17486 ?

Reply:
Where is the test for the issue solved here - RavenDB-21484?

Reply:
That tests is able to showcase both this and RavenDB-21525

Reply:
How it could showcase this case while there is no highlighting involved here?